backend/open_webui/models/notes.py

```
@@ -0,0 +1,346 @@
+            grants_map = AccessGrants.get_grants_by_resources("note", note_ids, db=db)
+            return [
+                self._to_note_model(
+                    note, access_grants=grants_map.get(note.id, []), db=db
+                )
+                for note in notes
+            ]
+
+    def search_notes(
+        self,
+        user_id: str,
+        filter: dict = {},
```

Ruff (0.15.1)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)